editor/editor_properties.cpp

```diff
@@ -2112,6 +2122,18 @@ void EditorPropertyResource::_menu_option(int p_which) {
 			TabContainer *tab_container = (TabContainer *)file_system_dock->get_parent_control();
 			tab_container->set_current_tab(file_system_dock->get_position_in_parent());
 		} break;
+		case OBJ_MENU_QUICK_LOAD: {
+
+			if (!quick_load) {
+				quick_load = memnew(EditorQuickOpen);
+				quick_load->connect("quick_load", this, "_quick_load");
+				add_child(quick_load);
+			}
+
+			quick_load->popup_dialog(base_type);
+			quick_load->set_title(TTR("Quick Load " + base_type + "..."));
```

Comment:
Should be `vformat(TTR("Quick Load %s..."), base_type)`

Reply:
corrected this right now!